docs/API_SPECS_SIMULATIONS.md

```diff
@@ -466,35 +472,55 @@ Content-Type: application/json; charset=utf-8
 ```
 ```json
 {
-  "ETag": "cec0722b205740",
+  "ETag": "969ee1fb277640",
   "Id": "1",
-  "Enabled": true,
-  "StartTime": "2019-02-03T14:00:00",
-  "EndTime": "2019-02-04T00:00:00",
+  "Name": "Sample Simulation",
+  "Description": "This is a sample simulation",
+  "Enabled": false,
+  "Running": false,
+  "IoTHubs": [
+      {
+          "ConnectionString": "default",
+          "PreprovisionedIoTHub": true,
+          "PreprovisionedIoTHubInUse": false,
+          "PreprovisionedIoTHubMetricsUrl": null
+      }
+  ],
+  "StartTime": "2018-08-31T00:30:00+00:00",
+  "EndTime": "2018-09-01T00:40:00+00:00",
+  "StoppedTime": "2018-08-31T00:35:00+00:00",
   "DeviceModels": [
-    {
-      "Id": "truck-01",
-      "Count": 3
-    },
-    {
-      "Id": "truck-02",
-      "Count": 1
-    },
-    {
-      "Id": "elevator-01",
-      "Count": 10
-    }
+      {
+          "Id": "truck-02",
+          "Count": 2
+      },
+      {
+          "Id": "elevator-02",
+          "Count": 4
+      }
   ],
+  "Statistics": {
+      "TotalMessagesSent": 60,
+      "AverageMessagesPerSecond": 6.43,
+      "FailedMessagesCount": 0,
+      "ActiveDevicesCount": 0,
+      "FailedDeviceConnectionsCount": 0,
+      "FailedDeviceTwinUpdatesCount": 0,
+      "SimulationErrorsCount": 0
+  },
   "$metadata": {
-    "$type": "Simulation;1",
-    "$uri": "/v1/simulations/1",
-    "$version": "1",
-    "$created": "2017-05-31T00:47:18+00:00",
-    "$modified": "2017-05-31T00:47:18+00:00"
+      "$type": "Simulation;1",
+      "$uri": "/v1/simulations/1",
+      "$created": "2018-08-31T00:30:00+00:00",
+      "$modified": "2018-08-31T00:30:00+00:00"
   }
 }
 ```
 
+There are some read-only properties in the api response. `Running` is a calculated property implying whether the simulation is currently running. The `StoppedTime` is the time when the simulation was manually stopped before it's scheduled `EndTime`. `PreprovisionedIoTHubMetricsUrl` implies whether the simulation is currently running using the `PreprovisionedIoTHub`. 'PreprovisionedIoTHubMetricsUrl' is the url to navigate to the IoT Hub metrics page. 
```

Comment:
split long line, so we can easily add comments in the PR and use IDE's split windows without using wordwrapping

Reply:
there are some errors in the text, here's a suggested edit:
```
* `Running` is a calculated property, reporting whether the simulation is
  currently running.
* `StoppedTime` is the time when the simulation was manually
  stopped before it's scheduled `EndTime`. 
* `PreprovisionedIoTHubInUse`: whether the simulation ran or is running
  using the Azure IoT Hub created during the solution deployment.
* 'PreprovisionedIoTHubMetricsUrl': the url to navigate to the IoT Hub
  metrics page. 
* `Statistics` is a set of read only properties providing counters
  at a given time. When the simulation is stopped, the current values
  for `TotalMessagesSent` and `AverageMessagesPerSecond` are retrieved from
  storage.
```